### ISSUE TITLE
Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml

### ISSUE THREAD

**Issue by sivabe35** [jayrambhia/Tooltip#14]
I set android:allowBackup to false and then I run my project it says below message

```
Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:24:9-36
	is also present at [com.fenchtose:tooltip:0.1.5] AndroidManifest.xml:12:9-35 value=(true).
	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:22:5-121:19 to override.
```

**Comment by jayrambhia** [comment]
As per the suggestion, you can add `tools:replace="android:allowBackup"` to your manifest. Read more here. **https://developer.android.com/studio/build/manifest-merge**.

I will fix this issue and release a new version in couple of days.

**Comment by jayrambhia** [comment]
Fixed in 0.1.6.
